test/extension/load_extension.test

```diff
@@ -53,11 +54,12 @@ QUACK
 statement error
 QUAC
 ----
-<REGEX>:.*Parser Error:.*
+<REGEX>:Parser Error: Did you.*
 
 statement error
 QUACK NOT QUACK
 ----
+<REGEX>:Parser Error: This is not.*
```

Comment:
I think this one should either be more specific or just be the `.*Parser Error.*` in it's current form it is not really testing the specific error, just the General error (parser error) and the existence of some words